modules/appeals_api/spec/services/appeals_api/evidence_submission_request_validator_spec.rb

```diff
@@ -61,43 +61,44 @@ module AppealsApi
         end
 
         it 'returns with :ok if the NOD was submitted less than 90 days prior' do
-          notice_of_disagreement = create(:notice_of_disagreement, board_review_option: 'evidence_submission')
+          notice_of_disagreement = create(:notice_of_disagreement_v2, :board_review_evidence_submission)
           create(:status_update,
                  to: 'submitted',
                  status_update_time: 89.days.ago,
                  statusable: notice_of_disagreement)
 
           result = described_class.new(
-            notice_of_disagreement.id, notice_of_disagreement.auth_headers['X-VA-SSN'], 'NoticeOfDisagreement'
+            notice_of_disagreement.id, notice_of_disagreement.auth_headers['X-VA-File-Number'], 'NoticeOfDisagreement'
           ).call
 
           expect(result).to eq([:ok, {}])
         end
 
         it 'returns an error if the veteran ISNT REAL' do
-          notice_of_disagreement = create(:notice_of_disagreement, board_review_option: 'evidence_submission')
+          notice_of_disagreement = create(:notice_of_disagreement_v2, :board_review_evidence_submission)
 
           result = described_class.new(
-            notice_of_disagreement.id, 'fake_ssn', 'NoticeOfDisagreement'
+            notice_of_disagreement.id, 'fake_file_number', 'NoticeOfDisagreement'
           ).call
 
           expect(result).to eq([
                                  :error,
                                  {
                                    title: 'unprocessable_entity',
-                                   detail: "Request header 'X-VA-SSN' does not match the associated appeal's SSN",
-                                   code: 'DecisionReviewMismatchedSSN',
+                                   detail: "Request header 'X-VA-SSN' or 'X-VA-File-Number' does not match the "\
```

Comment:
Again, this will change for both NOD v1 and v2, and we don't want to change responses on v1.

Reply:
K.